scripts/test-postgres-darwin.sh

```diff
@@ -32,13 +32,21 @@ tar -xJf $TRG_DIR/pg-dist/$ZIP_FILE -C $TRG_DIR/pg-test
 $TRG_DIR/pg-test/bin/initdb -A trust -U postgres -D $TRG_DIR/pg-test/data -E UTF-8
 $TRG_DIR/pg-test/bin/pg_ctl -w -D $TRG_DIR/pg-test/data -o '-p 65432 -F -c timezone=UTC -c synchronous_commit=off -c max_connections=300' start
 
+# Shutdown DB server and do cleanup on exit
+function cleanup() {
+  local errcode=$?
+  $TRG_DIR/pg-test/bin/pg_ctl -w -D $TRG_DIR/pg-test/data stop
+  rm -rf $TRG_DIR
+  return $errcode
+}
+
+trap cleanup EXIT
```

Comment:
Also a very smart solution 👍

Reply:
Thanks. Those 6 hours timeout bugged me until I found the cause :-)